src/main/java/com/codeborne/selenide/commands/closest/ClosestRuleEngine.java

```diff
@@ -0,0 +1,28 @@
+package com.codeborne.selenide.commands.closest;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class ClosestRuleEngine {
+
+  private static final List<ClosestRule> rules = new ArrayList<>();
```

Comment:
It may be simplified: just use `Arrays.asList(...)` instead of `new ArrayList<>()`